tests/TestGalaxyObjects.py

```diff
@@ -706,6 +741,40 @@ def test_params(self):
 
 
 @test_util.skip_unless_galaxy()
+class TestRunDatasetCollectionWorkflow(GalaxyObjectsTestBase):
+
+    def setUp(self):
+        super(TestRunDatasetCollectionWorkflow, self).setUp()
+        wf_file = os.path.join(THIS_DIR, 'data', 'dataset_collection_run.ga')
+        with open(wf_file) as f:
+            self.wf = self.gi.workflows.import_new(f.read())
+        self.hist_name = 'test_%s' % uuid.uuid4().hex
+        self.hist = self.gi.histories.create(self.hist_name)
+
+    def tearDown(self):
+        self.wf.delete()
+
+    @test_util.skip_unless_galaxy('release_14.06')
```

Comment:
I'd move `'release_14.06'` to the class decorator at line 743 and remove this line.